bin/git-utimes

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env bash
+#
+# Change files modification time to their last commit date
+#
+# Copyright (c) 2020 Vitaly Chikunov <vt@altlinux.org>.
+# SPDX-License-Identifier: MIT
+
+if [ "$1" = --touch ]; then
+  # Internal use option only just to parallelize things.
+  shift
+  for f; do
+    t=$(git --no-pager log --no-renames --pretty=format:%cI -1 @ -- "$f" 2>/dev/null)
+    if [ -n "$t" ]; then
+      echo "+ touch -d $t $f" >&2
+      touch -d "$t" "$f"
+    fi
+  done
+  exit
+else
+  # `-n` should be limited or parallelization will not give effect,
+  # because all args will go into single worker.
+  NPROC=$(nproc)
```

Comment:
That does not work on FreeBSD. See https://unix.stackexchange.com/questions/108821/number-of-cores-on-freebsd